src/terminology/path.md

```diff
@@ -14,7 +14,7 @@ In the example below, the highlighted path `/root/remi/head_M_hrc/GEO/head_M_hrc
 ![](../images/terminology/prim_path.png)
 ```
 
-As for properties, taking the previous example, inspecting the `points` attribute yields the path `/root/remi/head_M_hrc/GEO/head_M_hrc/eyeScrew_L_geo.points`. Property paths are constructed by appending the property name to a prim path, delimited by a `.` character.
+As for properties, taking the previous example, inspecting the `points` property yields the path `/root/remi/head_M_hrc/GEO/head_M_hrc/eyeScrew_L_geo.points`. Property paths are constructed by appending the property name to a prim path, delimited by a `.` character.
```

Comment:
I'm not sure how to feel about this. `points` is indeed a property, but it is specifically a property of the _attribute_ type. At this point in the book the difference between `attribute` and `relationship` and their parent moniker of `property` should be clear.

Reply:
Yeah, you're right; I only looked at it grammatically.
But we can expect the reader to know about the property-attribute connection since it is clearly stated at the beginning of the "USD Terminology" chapter.